src/main/java/com/coronation/nucleus/entities/Shareholder.java

```diff
@@ -32,16 +32,17 @@ public class Shareholder extends BaseEntity {
     @Column(name = "TOTAL_SHARES")
     private Long totalShares;
 
-    @Column(name = "PAR_VALUE")
-    private Long parValue;
+    @Column(name = "PRICE_PER_VALUE")
+    private Long pricePerShare;
```

Comment:
We agreed to remove this remember?

Reply:
No, I showed you something on the view remember